.github/workflows/main.yml

```diff
@@ -33,6 +33,7 @@ jobs:
       - name: Set up environment
         run: |
           pip install .
+          pip install types-jsonschema types-PyYAML  # Install type stubs
```

Comment:
These would also need to be added to the `.precommit.config.yaml` : https://github.com/aldbr/dirac-cwl-proto/blob/d513b1d204fab368167deddd01cbb5866d400200/.pre-commit-config.yaml#L30